UltimateWoltlabMaster.sh

```diff
@@ -0,0 +1,123 @@
+reset
+
+echo "UltimateWoltlabMaster von Justman10000"
+echo "Dies ist unter der GNU General Public License v3.0 lizenziert"
+
+echo "Was soll getan werden?"
+echo "Installieren [1]"
+echo "Deinstallieren [2]"
+echo "Zurück [3]"
+read WhatDo
+
+clear
+
+FileName=WoltlabSuite.zip
+
+case $WhatDo in
+    "1")
+        wget https://assets.woltlab.com/release/woltlab-suite-5.4.10.zip
```

Comment:
The version is hardcoded. It would be better to implement a check to always download the latest version of the WSC. Or alternatively, that you can choose a version yourself.